File: CS/TutorialsCustomItems/CustomItems/FunnelItemMetadata.cs
using System;
using System.ComponentModel;
using DevExpress.DashboardCommon;
using DevExpress.XtraCharts;

namespace TutorialsCustomItems{
    [DisplayName("Funnel"),
    Description("Funnel description"),
    CustomItemImage("TutorialsCustomItems.images.Funnel.svg")]
    public class FunnelItemMetadata : CustomItemMetadata {
        [DisplayName("Value"),
        EmptyDataItemPlaceholder("Value"),
        SupportColoring(DefaultColoringMode.None)]
        public Measure Value {
            get { return GetPropertyValue<Measure>(); }
            set { SetPropertyValue(value); }
        }
        [DisplayName("Arguments"),
        EmptyDataItemPlaceholder("My Argument"),
        SupportColoring(DefaultColoringMode.Hue),
        SupportInteractivity]
        public DimensionCollection Arguments { get; } = new DimensionCollection();

    }
}

File: VB/TutorialsCustomItems/CustomItems/FunnelItemMetadata.vb
Imports System
Imports System.ComponentModel
Imports DevExpress.DashboardCommon
Imports DevExpress.XtraCharts

Namespace TutorialsCustomItems
    <DisplayName("Funnel"), Description("Funnel description"), CustomItemImage("Funnel.svg")>
    Public Class FunnelItemMetadata
		Inherits CustomItemMetadata

		<DisplayName("Value"), EmptyDataItemPlaceholder("Value"), SupportColoring(DefaultColoringMode.None)>
		Public Property Value() As Measure
			Get
				Return GetPropertyValue(Of Measure)()
			End Get
			Set(ByVal value As Measure)
				SetPropertyValue(value)
			End Set
		End Property
		<DisplayName("Arguments"), EmptyDataItemPlaceholder("My Argument"), SupportColoring(DefaultColoringMode.Hue), SupportInteractivity>
		Public ReadOnly Property Arguments() As New DimensionCollection()

	End Class
End Namespace
